coniferest/onnx/_parse.py

```diff
@@ -0,0 +1,50 @@
+from ._container import ConiferestModelContainer
+from onnxconverter_common.topology import Topology
+from onnxconverter_common.data_types import FloatTensorType
+
+from ..aadforest import AADForest
+
+
+def _get_coniferest_operator_name(model):
+    if isinstance(model, AADForest):
+        return "AADForest"
+
+    raise ValueError("No proper operator name found for '%s'" % type(model))
+
+def _parse_coniferest(scope, model, inputs):
+    this_operator = scope.declare_local_operator(_get_coniferest_operator_name(model), model)
+    this_operator.inputs = inputs
+
+    # FIXME: probably another variable is required for anomality label
```

Comment:
Do we keep it for now?

Reply:
I think so. Really, we never need anomality labels, only the scores.